overlord/configstate/configcore/prompting.go

```diff
@@ -66,6 +116,17 @@ func doExperimentalApparmorPromptingDaemonRestart(c RunTransaction, opts *fsOnly
 			}
 			return fmt.Errorf("cannot enable prompting feature as it is not supported by the system: %s", whyNot)
 		}
+
+		handlers, err := findPromptingRequestsHandlers(st)
+		if err != nil {
+			return err
+		}
+
+		if len(handlers) == 0 {
+			return fmt.Errorf("cannot enable prompting feature no interfaces requests handler services are installed")
+		}
+
+		// TODO start handlers as a change and wait for completion
```

Comment:
Is this TODO for follow-up PR?

Reply:
yes

Reply:
this one: https://github.com/canonical/snapd/pull/14351